docs/resources/connection_database.md

```diff
@@ -0,0 +1,187 @@
+---
+page_title: "Resource: auth0_connection_database"
+description: |-
+  With Auth0, you can define sources of users, otherwise known as connections, which may include identity providers (such as Google or LinkedIn), databases, or passwordless authentication methods. This resource allows you to configure and manage connections to be used with your clients and users.
+---
+
+# Resource: auth0_connection_database
+
+With Auth0, you can define sources of users, otherwise known as connections, which may include identity providers (such as Google or LinkedIn), databases, or passwordless authentication methods. This resource allows you to configure and manage connections to be used with your clients and users.
```

Comment:
Admittedly, this description makes a lot less sense being so generic, but I don't think it's time to abstract the description out further.